terraform/modules/loadbalancer/data.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_route53_zone" "main" {
```

Comment:
Should be using remote state references

Reply:
Obviously better to use remote states but as the zone is dynamic based on the parent domain surely data makes more sense? This pattern is taken from aws-conourse https://github.com/dwp/aws-concourse/blob/master/terraform/modules/loadbalancer/data.tf

Reply:
As we are now using only one zone, this is now feasible